apps/aehttp/src/aehttpc_aeternity.erl

```diff
@@ -163,19 +199,45 @@ post_pin_tx(SignedSpendTx, NodeSpec) ->
     %lager:debug("PINNING: wrote to PC tx hash: ~p", [TxHash]),
     encode_child_pin_payload(TxHash).
 
+pin_contract_call(ContractPubkey, PinTx, Who, Amount, _Fee, SignModule) ->
+    Nonce = get_local_nonce(Who),
+    {ok, CallData} = aeb_fate_abi:create_calldata("pin", [{bytes, PinTx}]),
+    ABI = 3, % not really nice, what is the supported version of getting the latest ABI version
```

Comment:
not really nice indeed

Reply:
`?ABI_FATE_SOPHIA_1` from include file 
 -include_lib("aecontract/include/aeconstract.hrl").

Reply:
Fixed (the code first, your typo then ;) )

Reply:
I didn't even look here? @klercker asked how to do it properly and I explained how, and then he said he had fixed all things discussed??!